lib/gcloud/dns/zone.rb

```diff
@@ -360,6 +363,7 @@ def records options = {}
           fail ApiError.from_response(resp)
         end
       end
+      alias_method :find_record, :records
```

Comment:
That will return multiple records. Its not likely IMO, but possible with how the DNS API is set up.